clapper/clapper.ml

```diff
@@ -0,0 +1,7 @@
+module Clapper = struct
+
+  let punctuate (lst : string list) (sep : string) : string =
+    String.concat sep lst ^ sep
+
+  let add_claps (lst : string list) : string = punctuate lst " 👏 "
```

Comment:
Is it good practice always to have type signatures, even when they can be inferred?